com.apifox.apifox.json

```diff
@@ -0,0 +1,133 @@
+{
+    "app-id": "com.apifox.apifox",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "run.sh",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "24.08",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=x11",
+        "--filesystem=home",
```

Comment:
why does this need home?

Reply:
import file with .json,postman,.har and so on. and support export file.